src/commands/selectCommandTemplate.ts

```diff
@@ -38,7 +38,7 @@ export async function selectBuildCommand(context: IActionContext, folder: vscode
     );
 }
 
-export async function selectRunCommand(context: IActionContext, fullTag: string, interactive: boolean, exposedPorts?: PortBinding[]): Promise<VoidCommandResponse> {
+export async function selectRunCommand(context: IActionContext, imageId: string, interactive: boolean, exposedPorts?: PortBinding[]): Promise<VoidCommandResponse> {

```

Comment:
`selectRunCommand` could accept an `imageId` argument in addition to the existing `fullTag` argument.

Reply:
Done.

![image](https://github.com/microsoft/vscode-docker/assets/8070232/8ab41aa2-0c28-4e52-b802-0a9f5b1a44d4)